gui/src/system/SystemPages.tsx

```diff
@@ -11,6 +11,7 @@ import { python } from "@codemirror/lang-python";
 import { faTrash } from '@fortawesome/free-solid-svg-icons';
 import Fa from 'solid-fa';
 import { ServerResponse } from "http";
+import { P } from "@tauri-apps/api/event-41a9edf5";
```

Comment:
What is this import? The name is extremely non-specific and the end of the import seems to be some kind of commit hash? Was this a mistake?